pkg/web/metricsHandler_test.go

```diff
@@ -0,0 +1,145 @@
+package web
+
+import (
+	"testing"
+
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+
+	"github.com/aerogear/aerogear-metrics-api/pkg/mobile"
+	"github.com/stretchr/testify/assert"
+	mock "github.com/stretchr/testify/mock"
+)
+
+type MockMetricsService struct {
```

Comment:
general question, should there be a mock here.  Could we inject just a mock dao into the real service here giving us something closer to an e2e test that can run without the db

Reply:
I would agree with you on this. I think the kind of testing you're suggesting would give us a much greater ROI. However I'm conscious that this code was already reviewed and merged as part of #14. I'm hoping that this PR doesn't grow any larger than it already is. 

I would like to see a new issue/ticket created for e2e style testing. The original goal of this ticket was just the database parts but then two other PRs ended up being merged into this branch. (lesson learned - let's avoid doing that again)